src/getOperationFieldAliasMapping.ts

```diff
@@ -49,31 +49,34 @@ function setSelectionSetAliases({
 export function getOperationFieldAliasMapping(schema: string): {
   fieldNameByAliasPath: Map<string, string>;
 } {
-  const fieldNameByAliasPath = new Map<string, string>();
-  const fragmentFieldNameByAlias = new Map<string, Map<string, string>>();
   const rootNode = parse(schema);
-  for (const definition of rootNode.definitions) {
-    if (definition.kind === 'FragmentDefinition') {
-      const fragmentAliases = new Map<string, string>();
-      setSelectionSetAliases({
-        fieldNameByAliasPath: fragmentAliases,
-        fragmentFieldNameByAlias,
-        selectionSet: definition.selectionSet,
-        currentPath: '',
-      });
-      fragmentFieldNameByAlias.set(definition.name.value, fragmentAliases);
-    } else if (definition.kind === 'OperationDefinition') {
-      setSelectionSetAliases({
-        fieldNameByAliasPath,
-        fragmentFieldNameByAlias,
-        selectionSet: definition.selectionSet,
-        currentPath: '',
-      });
-    } else {
-      throw new Error(
-        `operation definitions contain non-operation definition of kind ${definition.kind}`,
-      );
```

Comment:
Now I just don't iterate over any kind that cannot be handled. But i am not sure about this change, since I am not fully aware of the consequences.